server/views/pages/vary/viewActive.njk

```diff
@@ -29,11 +29,11 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
             {% if licence.typeCode === 'AP' or licence.typeCode === 'AP_PSS' %}
-                {{ additionalLicenceConditions(licence) }}
-                {{ bespokeConditions(licence) }}
+                {{ additionalLicenceConditions(licence, null, [], 'Additional licence conditions') }}
+                {{ bespokeConditions(licence, null, 'Bespoke licence conditions') }}
             {% endif %}
             {% if licence.typeCode === 'PSS' or licence.typeCode === 'AP_PSS' %}
-                {{ additionalPssConditions(licence) }}
+                {{ additionalPssConditions(licence, null, [], 'Additional post sentence supervision requirements') }}
```

Comment:
these macros are used for non-vary journeys too and a title change for title in non-vary  journeys hasn't been requested .